assets/timber.js.liquid

```diff
@@ -243,8 +245,11 @@ timber.productPage = function (options) {
       $comparePrice
         .html({{ 'products.product.compare_at' | t | json }} + ' ' + Shopify.formatMoney(variant.compare_at_price, moneyFormat))
         .show();
+      $comparePriceA11y.attr('aria-hidden', 'false');
+      $priceA11y.attr('aria-hidden', 'false');
```

Comment:
My understanding is that `$priceA11y` should never be set to `aria-hidden="true"` as the product price should always be available to the screen reader/VO, is that correct?

Setting `aria-hidden` to `false` may not be needed though, would not setting any value to the `aria-hidden` attribute have the same effect as setting it to `false` here?